docs/eventing/sink/_index.md

```diff
@@ -44,6 +44,35 @@ spec:
       name: my-kafka-sink
 ```
 
+## Setting up a Custom Resource to be used as a sink
+To allow a Custom Resource to act as a sink for events, there are two things needed: 
+
+#### 1. Make the resource Addressable
+To make a Custom Resource Addresable, it needs to contain a `status.address.url`. More information about [Addressable resources](https://github.com/knative/specs/blob/main/specs/eventing/interfaces.md#addressable).
+
+#### 2. Create an addressable-resolver ClusterRole
+An addressable-resolver ClusterRole is needed in order to obtain the necessary RBAC rules for the sink to receive events.
+```yaml
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: kafkasinks-addressable-resolver
```

Comment:
maybe say something about `kafkasinks` being an example here and it should be replaced by the name of your CRD sink?